flow/project.py

```diff
@@ -2312,11 +2329,37 @@ def hello(job):
         cls._OPERATION_FUNCTIONS.append((name, func))
         return func
 
+    @classmethod
+    def _collect_operations(cls):
+        "Collect all operations that were add via decorator."
+        operations = []
+        for cls_ in cls.__mro__:
```

Comment:
Can we name this to something like `parent_class` rather than just `cls_`? (Same for the below functions).